### DISCUSSION TITLE
我在RoboMaster那些年

### DISCUSSION THREAD

**Discussion by giscus** [caibingcheng/comment#310] [Announcements]
# 我在RoboMaster那些年

昨晚和朋友聊到扫地机器人、无人机、视觉，就想起来“年轻”时候参加的RoboMaster机甲大师赛，因此想分享一下参赛那年的经历。

https://imcbc.cn/202301/robomaster-experience/

**Comment by penndu** [comment]
记得还有个机器人的电视剧！

**Reply by caibingcheng** [reply]
不会是铁甲小宝吧🤖

**Reply by penndu** [reply]
我怎么回？

**Comment by penndu** [comment]
辽宁联通无法访问GitHub，来评论还得现开代理~

**Comment by ryan4yin** [comment]
看着好有意思啊，玩起来~
